app/src/main/java/be/digitalia/mediasession2mqtt/MainWorker.kt

```diff
@@ -110,25 +107,30 @@ class MainWorker @Inject constructor(
             val device: DeviceInfo
         )
 
-        // TODO: using a data class for storing sensor metadata would be nicer and let us configure
-        //  icons etc. but this suffices for a proof-of-concept.
-        val sensors = mapOf(
-            "Application ID" to "$APPLICATION_ID_SUB_TOPIC",
-            "Playback state" to "$PLAYBACK_STATE_SUB_TOPIC",
-            "Media title" to "$MEDIA_TITLE_SUB_TOPIC"
+
+        val deviceInfo = DeviceInfo(
+            name = "MediaSession2MQTT",
+            manufacturer = Build.MANUFACTURER,
+            model = Build.MODEL,
+            identifiers = listOf(deviceId.toString())
         )
 
-        val deviceInfo = DeviceInfo()
+
+        val sensors = listOf(
+            Pair("Application ID", "$APPLICATION_ID_SUB_TOPIC"),
```

Comment:
You don't need to use String interpolation if you want to reuse an existing String as-is. I missed this during the first review.
`"$APPLICATION_ID_SUB_TOPIC"` -> `APPLICATION_ID_SUB_TOPIC`

Reply:
Done.